docker/prod/alerting.yaml

```diff
@@ -28,3 +28,11 @@ spec:
       annotations:
         description: CloseWikiMaintenance maintenance script did not run in the last 48 hours
         summary: CloseWikiMaintenance last succeeded {{humanizeDuration $value}} ago
+    # DumpStarters should be run every 24h
+    - alert: mediawiki_maintenance_scripts_DumpStarters_last_run
+      expr: time() - max(mediawiki_mediawiki_maintenance_scripts_last_success{env="prod", script_class="DumpStarters"}) > 3600 * 25
```

Comment:
why not 86400 as in other alert?

Reply:
`CloseWikiMaintenance` can take a while (a few hours), while `DumpStarters` runs within a minute. So this check is basically a day plus a bit (~25 hours).